test/Integration/SqlOutputBindingIntegrationTests.cs

```diff
@@ -49,7 +49,7 @@ public void AddProductTest(int id, string name, int cost, SupportedLanguages lan
         [SqlInlineData(-500, "ABCD", 580)]
         // Currently Java functions return null when the parameter for name is an empty string
         // Issue link: https://github.com/Azure/azure-functions-sql-extension/issues/517
-        [UnsupportedLanguages(SupportedLanguages.PowerShell, SupportedLanguages.Java)]
+        [UnsupportedLanguages(SupportedLanguages.Java)]
```

Comment:
Was looking at the unsupported PowerShell found that this one must have gotten reverted on a PR: https://github.com/Azure/azure-functions-sql-extension/pull/518.